### DISCUSSION TITLE
Notification template or example?

### DISCUSSION THREAD

**Discussion by VideoScape** [upptime/upptime#903] [General] [answered]
Hey there I am trying to setup the SMTP and discord notification for it but I am not sure where they go in the file.

I have tried a few different ways but I am not getting anything or any errors either so I am not sure what I am doing wrong.

Could I get a working example? I would really appreciate it!

Thanks!

**Comment by AnandChowdhary** [comment, marked as answer]
These are the environment variables you need to set for SMTP: https://upptime.js.org/docs/notifications/#smtp. On the same page, you'll find environment variables for Discord as well. You have to set these as Repository secrets for Actions; you'll see a screenshot of those at the top of the linked page. Let me know if you have any trouble!
